blocks/library/text/index.js

```diff
@@ -47,7 +47,25 @@ registerBlock( 'core/text', {
 		}
 	],
 
-	edit( { attributes, setAttributes, insertBlockAfter, focus, setFocus } ) {
+	formatting: [
```

Comment:
Also, if we're introducing changes to the block API, we should probably document them at the same time. Specifically thinking about available `format` options.